core/src/main/java/org/keycloak/crypto/AsymmetricSignatureSignerContext.java

```diff
@@ -27,6 +27,10 @@ public AsymmetricSignatureSignerContext(KeyWrapper key) throws SignatureExceptio
         this.key = key;
     }
 
+    public KeyWrapper getKey() {
```

Comment:
I believe getKey() is not a necessary addition.

Reply:
It's used from ServerECDSASignatureSignerContext

Reply:
You probably meant getKey() method in AsymmetricSignatureVerifierContext.java where it can be indeed removed.

Reply:
No, I meant this addition. Please see comments in `ServerECDSASignatureSignerContext`.